lib/types/entry.types.ts

```diff
@@ -6,6 +6,7 @@ type TaskState = 'active' | 'resolved'
 
 export interface TaskInputData {
   assignedToId: string
+  assignedToType?: string
```

Comment:
can we limit the type to only `User` or `Team` right now? Also better DX for auto completion

Reply:
Good point.